x-pack/plugins/triggers_actions_ui/public/application/sections/actions_connectors_list/components/actions_connectors_list.tsx

```diff
@@ -436,12 +467,20 @@ export const ActionsConnectorsList: React.FunctionComponent = () => {
           setAddFlyoutVisibility={setAddFlyoutVisibility}
         />
         {editedConnectorItem ? (
-          <ConnectorEditFlyout
-            key={editedConnectorItem.id}
-            initialConnector={editedConnectorItem}
-            editFlyoutVisible={editFlyoutVisible}
-            setEditFlyoutVisibility={setEditFlyoutVisibility}
-          />
+          <>
```

Comment:
It's generally suggested to use `Fragment` instead of empty nodes, as React can optimise around these easier.

```suggestion
          <Fragment>
```